KPP/custom/CMakeLists.txt

```diff
@@ -1,5 +1,32 @@
 # KPP/custom/CMakeLists.txt
 
+#----------------------------------------------------------------------------
+# Determine KPP integrator name and set corresponding C-preprocessor flags.
+# This enables integrator-specific handling in the custom mechanism.
+#----------------------------------------------------------------------------
+
+# Get integrator name
+EXECUTE_PROCESS(
+  COMMAND grep \#INTEGRATOR ${CMAKE_CURRENT_SOURCE_DIR}/custom.kpp
+  OUTPUT_VARIABLE RESULT
+)
+separate_arguments(SUBSTRINGS UNIX_COMMAND "${RESULT}")
+list(GET SUBSTRINGS 1 KPP_INT_NAME)
```

Comment:
Same comment as before but for KPP_INT_NAME. I prefer writing out INTEGRATOR so it is not confused with integer.

Reply:
Now fixed in commit [8c306d5](https://github.com/geoschem/geos-chem/pull/2689/commits/8c306d5957d41d703677a58090bf44c53b2e10ee)